package.json

```diff
@@ -41,6 +41,7 @@
 		"integration-tests/import",
 		"integration-tests/server",
 		"modules/components",
+		"modules/charts",
 		"modules/server"
```

Comment:
this is more than ok for now, but "components" seems too generic now, and we should all have a chat to figure out next steps here (e.g. thinking of lectern-ui)